doc/rados/operations/pools.rst

```diff
@@ -94,19 +94,12 @@ To get even more information, you can execute this command with the ``--format``
 Creating a Pool
 ===============
 
-Before creating a pool, consult `Pool, PG and CRUSH Config Reference`_.  Your
-Ceph configuration file contains a setting (namely, ``pg_num``) that determines
-the number of PGs.  However, this setting's default value is NOT appropriate
-for most systems.  In most cases, you should override this default value when
-creating your pool.  For details on PG numbers, see `setting the number of
-placement groups`_
-
-For example:
-
-.. prompt:: bash $
-
-    osd_pool_default_pg_num = 128
-    osd_pool_default_pgp_num = 128
+Before creating a pool, consult `Pool, PG and CRUSH Config Reference`_. The
+Ceph central configuration database in the monitor cluster contains a setting
+(namely, ``osd_pool_default_pg_num``) that determines the number of PGs per
+pool. It is possible to change this value from its default. For more on the
```

Comment:
when a pool is created without explicitly specifying a per-pool value.  It is ...

Reply:
Mind you I always counsel people to specify `pg_num` when creating a pool.  The party line is to now let the autoscaler handle this, but heterodoxy is fairly common.

Reply:
> when a pool is created without explicitly specifying a per-pool value. It is ...

Accepted.

> Mind you I always counsel people to specify `pg_num` when creating a pool. The party line is to now let the autoscaler handle this, but heterodoxy is fairly common.

I have a preference here, which is to push the party line of letting the autoscaler handle pool allocation during creation. However, if there are good reasons to forego the autoscaler, I'd rather they appear in the official Ceph documentation instead of on StackOverflow or somewhere like that. The whole issue of pool allocation seems to me to have several threads, which haven't been carefully untangled in the docs. Anyway, this PR is one of what will certainly be a series of pool-documentation-related PRs. All of this material is at this time even more malleable than it normally is. I am open to capturing in the documentation as much pool-allocation expertise as the upstream community is willing to throw at me.